client/data/marketplace/use-es-query.ts

```diff
@@ -41,29 +41,28 @@ const mapIndexResultsToPluginData = ( results: ESHits ): Plugin[] => {
 	}
 	return results.map( ( { fields: hit, railcar } ) => {
 		const plugin: Plugin = {
-			name: hit.plugin.title, // TODO: add localization
+			name: hit.plugin?.title, // TODO: add localization
```

Comment:
Are we intentionally leaving all those fields `undefined` when `hit.plugin` is not there? Should we default to an empty string or `undefined` will be fine?

Reply:
Yes, it is intentional. We mount the `fullPlugin` object with values from different sources, we want it to be `undefined` in order to not override any real value.


https://github.com/Automattic/wp-calypso/blob/440704bae9da87a7c73de7536e25bc26fc3341e0/client/my-sites/plugins/plugin-details.jsx#L167-L175